app/javascript/stylesheets/application.scss

```diff
@@ -116,6 +119,21 @@ a:hover {
   background-image: linear-gradient(62deg, $gradation-p1-1 0%, $gradation-p1-2 100%);
 }
 
+card.text {
+  color: $default-button-letter;
+  font-weight: bold;
+}
+
+.card-header {
+  background-color: $card-header-color;
+}
+
+.card-header a {
+  color: $default-button-letter;
+  font-weight: bold;
+  text-decoration:none;
```

Comment:
__[SCSS-Lint]__

Colon after property should be followed by one space


<!-- Comment by [SideCI](https://sideci.com) -->